src-java/blue-green/src/main/java/org/openkilda/bluegreen/ZkWriter.java

```diff
@@ -51,11 +48,21 @@ public void setState(int active) {
         }
     }
 
+    @Override
+    public void init() {
+        try {
+            initZk();
+            validateNodes();
```

Comment:
`validateNodes()` will never be called, if initZk() on previous line raise exception.

Reply:
And what is the problem? `nodesValidated` wouldn't be set to true? If yes it exactly what I want. To return false in `isCOnnectionAlive` in this case